### DISCUSSION TITLE
How to get syntax-highlighting for emphasis in markdown?

### DISCUSSION THREAD

**Discussion by tevaughan** [NvChad/NvChad#2555] [Q&A] [answered]
When I display a markdown-file, headings are nicely colored, and I can even get the right C++-highlighting within a code-block marked as C++.

However, the inline emphasis seems not to be highlighted. For example, neither single nor double asterisks produce any highlighting for me, regardless of theme.

Is this normal?  If so, is there a work-around? If not, how do I figure out what's wrong?

**Comment by siduck** [comment, marked as answer]
@tevaughan not a nvchad issue, ask in r/neovim
